src/app/pages/activation/activation.component.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © Live-Poll 2020-2021. All rights reserved
+ */
+
+import {Component, EventEmitter, OnInit} from '@angular/core';
+import {Poll} from '../../model/poll';
+import {User} from '../../model/user';
+import {ActivatedRoute} from '@angular/router';
+import {AccountService} from '../../service/account.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './activation.component.html',
+  styleUrls: ['./activation.component.sass']
+})
+export class ActivationComponent implements OnInit {
+
+  loading = true;
+  success = true;
+
+  /**
+   * Initialize component
+   *
+   * @param route Active route
+   * @param accountService Injected AccountService
+   */
+  constructor(
+    private route: ActivatedRoute,
+    private accountService: AccountService
+  ) {}
+
+  ngOnInit(): void {
+    this.route.params.subscribe(params => {
+      this.accountService.confirm(params.token).subscribe((_) => {
```

Comment:
`(_)` kind of bugs me 😅 Maybe find a way to tell our code inspector to ignore this one. I see no benefit in wrapping an underscore in parantheses, it just adds another step for my brain to ignore these parentheses when reading the code and thus makes it harder to understand (although only slightly).